functional/FunctionalTest.java

```diff
@@ -3593,37 +3596,22 @@ public static void setDefaultRetention_test() throws Exception {
       try {
         ObjectLockConfiguration config =
             new ObjectLockConfiguration(RetentionMode.COMPLIANCE, new RetentionDurationDays(10));
-        client.setDefaultRetention(bucketName, config);
+        client.setDefaultRetention(
+            SetDefaultRetentionArgs.builder().bucket(bucketName).config(config).build());
       } finally {
         client.removeBucket(RemoveBucketArgs.builder().bucket(bucketName).build());
       }
-      mintSuccessLog("setDefaultRetention (String bucketName)", null, startTime);
+      mintSuccessLog(methodName, null, startTime);
```

Comment:
Retention mode and duration can be logged as args

Reply:
done